todolist/src/App.css

```diff
@@ -1,38 +1,34 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+body {
+  background: linear-gradient(135deg, #8e2de2, #4a00e0, #00c6fb);
+  background-size: 200% 200%;
+  animation: gradientAnimation 15s ease infinite;
+  justify-content: center;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.app {
   display: flex;
   flex-direction: column;
-  align-items: center;
+  height: 75vh;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: center;
+  /*
+  background-color:#272727;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+  border-radius: 8px;
+  margin:0 auto;
+  animation: fadeIn 1s ease; */
 }
 
-.App-link {
-  color: #61dafb;
+.app h1 {
+  text-align: center;
+  margin-bottom: 24px;
 }
 
-@keyframes App-logo-spin {
+@keyframes fadeIn {
   from {
-    transform: rotate(0deg);
+    opacity: 0;
   }
   to {
-    transform: rotate(360deg);
+    opacity: 1;
   }
-}
+}
```

Comment:
Configure VS code to automaticallyy add a new line at the end of file

Reply:
Done.